src/Manifolds.jl

```diff
@@ -108,15 +108,15 @@ Abstract type for defining statistical estimation methods.
 """
 abstract type AbstractEstimationMethod end
 
-@doc doc"""
+@doc raw"""
     hat(M::Manifold, x, vⁱ)
 
 Given a basis $e_i$ on the tangent space at a point $x$ and tangent
 component vector $v^i$, compute the equivalent vector representation
 $v=v^i e_i$, where Einstein summation notation is used:
 
 ````math
-\wedge: v^i \mapsto v^i e_i
+∧ : v^i ↦ v^i e_i
```

Comment:
Let's adopt the tangent vector notation `X` and coefficients `X^i` here and for the variable names. Also for `vee`.

Reply:
Yes, I basically just missed to rename these two functions parameters.